NuRadioMC/EvtGen/generator.py

```diff
@@ -784,23 +789,50 @@ def generate_eventlist_cylinder(filename, n_events, Emin, Emax,
         the minimum neutrino energy
     Emax: float
         the maximum neutrino energy
-
-    fiducial_rmin: float
-        lower r coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-    fiducial_rmax: float
-        upper r coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-    fiducial_zmin: float
-        lower z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-    fiducial_zmax: float
-        upper z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
-    full_rmin: float (default None)
-        lower r coordinate of simulated volume (if None it is set to 1/3 of the fiducial volume, if second vertices are not activated it is set to the fiducial volume)
-    full_rmax: float (default None)
-        upper r coordinate of simulated volume (if None it is set to 5x the fiducial volume, if second vertices are not activated it is set to the fiducial volume)
-    full_zmin: float (default None)
-        lower z coordinate of simulated volume (if None it is set to 1/3 of the fiducial volume, if second vertices are not activated it is set to the fiducial volume)
-    full_zmax: float (default None)
-        upper z coordinate of simulated volume (if None it is set to 5x the fiducial volume, if second vertices are not activated it is set to the fiducial volume)
+    volume: dict
+        a dictionary specifying the simulation volume
+        can be either a cylinder spefified via the keys
+            * fiducial_rmin: float
+                lower r coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
+            * fiducial_rmax: float
+                upper r coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
+            * fiducial_zmin: float
+                lower z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
+            * fiducial_zmax: float
+                upper z coordinate of fiducial volume (the fiducial volume needs to be chosen large enough such that no events outside of it will trigger)
+            * full_rmin: float (optional)
+                lower r coordinate of simulated volume (if None it is set to 1/3 of the fiducial volume, if second vertices are not activated it is set to the fiducial volume)
+            * full_rmax: float (optional)
+                upper r coordinate of simulated volume (if None it is set to 5x the fiducial volume, if second vertices are not activated it is set to the fiducial volume)
```

Comment:
These doc strings must be changed as well.

Reply:
done